.github/workflows/nightly.yaml

```diff
@@ -96,6 +118,33 @@ jobs:
             TAG=${{ steps.meta.outputs.version }}
             COMMIT=${{ github.sha }}
             REPO_INFO=https://github.com/${{ github.repository }}.git
+      - name: Build and push Red Hat Scan Registry image
+        id: docker_build_redhat_scan_registry
+        env:
+          TAG: ${{ steps.meta_redhat_scan_registry.outputs.version }}
+        uses: docker/build-push-action@v3.1.1
+        with:
+          push: true
+          file: Dockerfile
+          tags: ${{ steps.meta_redhat_scan_registry.outputs.tags }}
+          cache-from: type=local,src=/tmp/.buildx-cache
+          target: redhat
```

Comment:
Given that this is expanding our "build image" CI part I think we can invest some time (not in this PR) to separate pipelines so that a generic docker images (for multiple architectures) are built in one branch of the pipeline, RH images (for architectures supported in RH) are built in a different branch and so on.

So that we both get the benefit of parallelized pipelines and separation of concerns (we'll clearly see which part of the build failed and only use secrets and envs needed for particular pipeline).

Do you mind creating an issue for something like this?

Reply:
I was actually wondering do we need at all a RedHat images that are pushed to dockethub.

Reply:
🤔 Not sure. Let's bring this up on today's sync, hm?